scalafmt-core/shared/src/main/scala/org/scalafmt/rewrite/RedundantBraces.scala

```diff
@@ -124,6 +124,9 @@ class RedundantBraces(ftoks: FormatTokens) extends FormatTokensRewrite.Rule {
             (ConvertToNewScala3Syntax.enabled ||
               !x.tokens.exists(_.is[Token.RightArrow])) =>
         removeToken
+      case p: Importer
```

Comment:
you need to put it in the case above, it also matches single-importee importers but mistakenly thinks that only rename and unimport are possible. something like this:
```
case meta.Importer(_, List(x))
  if !(x.is[Importee.Rename] || x.is[Importee.Unimport) ||
    ...
```

also, as you can see, we are not using any parameters to rewrite this, so my opinion is that we don't need a new parameter.

Reply:
Correct, I've added it to the one above and added (and fixed) some extra tests for the name scenario's